kalite/distributed/static/js/distributed/intro-injector.js

```diff
@@ -0,0 +1,122 @@
+//on page load
+$(function() {
+  console.log( "ready!" );
+
+  //Use relative URL to determine which YAML file to parse
```

Comment:
I know I helped you write the comments, but you should change them to reflect what's actually going on. Namely there's no YAML around these parts -- that should all be handled on the server-side.